### DISCUSSION TITLE
Startup message box to inform about changes

### DISCUSSION THREAD

**Discussion by luzip665** [exaile/exaile#873] [Ideas]
What do you think about a startup message box to inform old users about important changes?
In special I would like to mention #871 and #836 because the user can decide what to do. But may be the user wouldn't do anything because of not knowing the new possibilities.
My idea comes from #682

I would say the message box should only be shown at the first startup after an update. The user could decide to keep or decline the box.

In case of a first run after a blank installation I wouldn't show the box. Instead the updated tag mapping should already be used.

**Comment by sjohannes** [comment]
Yeah, I think that makes sense.
